freqtrade/freqtradebot.py

```diff
@@ -938,8 +938,7 @@ def _safe_sell_amount(self, pair: str, amount: float) -> float:
         """
         # Update wallets to ensure amounts tied up in a stoploss is now free!
         self.wallets.update()
-
-        wallet_amount = self.wallets.get_free(pair.split('/')[0])
+        wallet_amount = self.wallets.get_free(self.exchange.get_pair_base_currency(pair))
```

Comment:
```suggestion
        trade_base_currency = self.exchange.get_pair_base_currency(pair)
        wallet_amount = self.wallets.get_free(trade_base_currency)
```
-- this `trade_base_currency` can later be used for logging; code unified with that used in `get_real_amount` below...

Reply:
I distrust changes done via "commit suggestion" directly when it touches code.

The above example - 1 typo - and everything fails ;) thats why i often prefer to do the change "manually" - offline in an editor ...

Reply:
Agree, but this only an easy format to "suggest" the change. you do not have to commit the suggestion, it can be done, yes, manually with the checks before committing...
👍